dashboard/app/models/user.rb

```diff
@@ -200,7 +200,7 @@ class User < ActiveRecord::Base
     class_name: 'Pd::Application::ApplicationBase',
     dependent: :destroy
 
-  after_create :associate_with_potential_pd_enrollments
+  after_create :associate_with_potential_pd_enrollments, :update_share_setting
```

Comment:
I'm fairly certain this won't actually commit the changes to the db. [after_create](https://apidock.com/rails/v1.2.0/ActiveRecord/Callbacks/after_create) runs after `save`, and setting the attribute on this instance (as `update_share_setting` does) will only affect the in-memory instance.

Modify tests to `.reload` after creation to be sure. I think this should be `before_create` instead of `after_create`